app/views/pages/about.html.erb

```diff
@@ -196,7 +196,7 @@
       <div class="main">
         <h2>
 			Where we band together to build
-			products people love.
+			products people use & love.
```

Comment:
I'm not sure about the ampersand vs. "and" -- @awwstn, what's our copy style guide say?